Detectors/TPC/simulation/macro/laserTrackGenerator.C

```diff
@@ -0,0 +1,44 @@
+class LaserTrackGenerator : public FairGenerator
+{
+ public:
+  LaserTrackGenerator() : FairGenerator("TPCLaserTrackGenerator")
+  {
+    mLaserTrackContainer.loadTracksFromFile();
+  }
+
+  Bool_t ReadEvent(FairPrimaryGenerator* primGen) override
+  {
+
+    // loop over all tracks and add them to the generator
+    const auto& tracks = mLaserTrackContainer.getLaserTracks();
+
+    // TODO: use something better instead. The particle should stop at the inner field cage of the TPC.
+    //       perhaps use a custom particle with special treatment in Detector.cxx
+    //const int pdgCode = 2212;
+    const int pdgCode = 11;
+    std::array<float, 3> xyz;
+    std::array<float, 3> pxyz;
+
+    for (const auto& track : tracks) {
+      track.getXYZGlo(xyz);
+      track.getPxPyPzGlo(pxyz);
+      // rescale to 10GeV to avoid segfault in geant
```

Comment:
This is ok (though 1 TeV not 10 GeV), just for the reference: there is a method ``o2::track::TrackParCov::invert()`` which makes the track to propagate in opposite direction.

Reply:
Yes! I updated the comment.
Thanks for the hint with the invert. I'll leave it as it is now, since I anyhow need to rescale.